client/reader/stream/index.jsx

```diff
@@ -437,15 +471,58 @@ class ReaderStream extends Component {
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 				/>
 			);
+			const sidebarContent = <ReaderSidebarFollowedSites path={ path } />;
+
+			if ( excludesSidebar.includes( streamType ) ) {
+				body = bodyContent;
+			} else if ( wideDisplay ) {
+				body = (
+					<div className="stream__two-column">
+						{ bodyContent }
+						<div className="stream__right-column">{ sidebarContent }</div>
+					</div>
+				);
+				baseClassnames = classnames( 'reader-two-column', baseClassnames );
+			} else {
+				body = (
+					<>
+						<div className="stream__header">
+							<SectionNav selectedText={ this.state.selectedTab }>
+								<NavTabs label="Status">
```

Comment:
Should this be localized?

```suggestion
								<NavTabs label={ translate( 'Status' ) }>
```